Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Hole_filling/Triangulate_hole_polyline.h

```diff
@@ -1106,9 +1106,10 @@ class Triangulate_hole_polyline_DT
     
     if(W.get(0, n-1) == Weight::NOT_VALID()) {
       #ifndef CGAL_TEST_SUITE
-      CGAL_warning(!"Returning no output. Filling hole with extra triangles is not successful!");
+      CGAL_warning(!"Returning no output using Delaunay triangulation.\n Falling back to the general Triangulation framework.");
```

Comment:
is the message Delaunay specific in case `CGAL_HOLE_FILLING_DO_NOT_USE_DT3` is defined?

Reply:
If `CGAL_HOLE_FILLING_DO_NOT_USE_DT3` is defined, `internal::Triangulate_hole_polyline_DT` is not used, so this message is not reached/printed